godtools/App/Features/Dashboard/Presentation/Dashboard/DashboardView.swift

```diff
@@ -17,41 +17,25 @@ struct DashboardView: View {
     init(viewModel: DashboardViewModel) {
         self.viewModel = viewModel
         
-        if #available(iOS 16.0, *) { } else {
-            UITabBar.appearance().isHidden = true
-        }
+        // Note: In iOS 16+, the preferred way to hide the tab bar is through the view modifier `.toolbar(.hidden, for: .tabBar)`, but there seems to be a bug where it only hides on the tab's first appearance.  May want to reinvestigate this in future versions.
+        UITabBar.appearance().isHidden = true
```

Comment:
Just to be safe it would be good to use the ```whenContainedInInstancesOf``` argument to restrict this to the DashboardView.

https://developer.apple.com/documentation/uikit/uiappearance/1615013-appearance